Makefile

```diff
@@ -230,6 +231,14 @@ gofmt: $(GOIMPORTS) add-headers
 autopep8: $(MAGE)
 	@$(MAGE) pythonAutopep8
 
+.PHONY: lint
+lint: fmt
+	@if [ -n "$(GIT_STATUS)" ]; then\
```

Comment:
is GIT_STATUS set before running the goal?

Reply:
I said nothing

Reply:
I think it's lazy evaluated, right?

Reply:
`make check` (which calls `mage check`) already checks for uncommitted changes. This is redundant. I think what's missing is the `fmt` dependency.

Reply:
I'll remove the lint target then, delegating it to check